### ISSUE TITLE
Cannot read property 'IsAndroid' of null

### ISSUE THREAD

**Issue by bekranker** [zmxv/react-native-sound#854]
:beetle: **Description**

I am tried to implement this package and it give me that error when I try to implement it.

<img width="443" alt="Screenshot 2024-12-31 at 21 30 07" src="https://github.com/user-attachments/assets/1efa7ec5-fde4-48fa-a2f2-46237ee04c7f" />

:beetle: **Please post your code:**
```js
import Sound from "react-native-sound";
```

:bulb: **Does the problem have a test case?**

**Is your issue with...**

- ✅ iOS
- [ ] Android
- [ ] Windows

**Are you using...**

- [ ] React Native CLI (e.g. `react-native run-android`)
- ✅ Expo
- [ ] Other: (please specify)

**Which versions are you using?**

- React Native Sound: 0.11.2
- React Native: 0.74.5
- iOS: 18.0
- Expo: 52.0.15

**Does the problem occur on...**

- ✅ Simulator
- [ ] Device

**If your problem is happening on a device, which device?**

- Device: IOS simulator in expo

**Comment by etroynov** [comment]
Looks like it not support new Arch:
https://github.com/zmxv/react-native-sound/pull/850

**Comment by Okelm** [comment]
No, it doesn't support new arch. @bekranker It means you can either go and use this version and test it (preferable) https://github.com/zmxv/react-native-sound/pull/850 or use older expo version
